packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/functions/AnyFunction.kt

```diff
@@ -122,4 +124,9 @@ abstract class AnyFunction(
   fun getCppRequiredTypes(): List<ExpectedType> {
     return desiredArgsTypes.map { it.getCppRequiredTypes() }
   }
+
+  @Suppress("FunctionName")
+  fun Enumerable(isEnumerable: Boolean = true) = apply {
```

Comment:
Honestly I'm more in favor of lowercased names for modifiers like this. We already have `.runOnQueue`, and both SwiftUI and Jetpack Compose are following this rule. What do you think?